R/utils.R

```diff
@@ -175,7 +175,11 @@ write_pbp <- function(seasons, dbConnection, tablename){
     if (!DBI::dbExistsTable(dbConnection, tablename)){
       pbp <- dplyr::bind_rows(default_play, pbp)
     }
-    DBI::dbWriteTable(dbConnection, tablename, pbp, append = TRUE)
+    if (DBI::dbExistsTable(dbConnection, tablename)) {
```

Comment:
This seems to be an issue of the DB driver rather than anything else to be honest. Every other driver handles this fine so I'd suggest filing an issue in the driver api repo

Reply:
bigrquery implements the dbWriteTable method here 
https://github.com/r-dbi/bigrquery/blob/73781ba90477025653ef2d10ee825bee7f407133/R/dbi-connection.R#L146-L193
It sets write_disposition to "WRITE_APPEND" if `append = TRUE`
<img width="857" alt="image" src="https://user-images.githubusercontent.com/38586519/196876216-dd361de3-d0f0-4254-9510-2852310bd0c5.png">
It is possible that Big Query errors if the table doesn't exist so this should be an issue in bigrquery. I will not change the nflfastR source to work around a bug in a DBI driver.